### PR TITLE
Add animations to CardsView and fix a couple of previews

### PR DESCRIPTION
**Pull request by RomaJZ**
https://user-images.githubusercontent.com/35544719/224725179-ce76155f-8d6e-4c06-bc13-6b79f524137d.mp4

### CODE REVIEW

**Comment by artem-y-eleks** [comment]
I believe the card at the bottom should be face-down?  
your_cards_bottom

File: iosApp/iosApp/Screens/Cards/CardsView.swift [hunk 1]
```diff
@@ -14,50 +14,115 @@ struct CardsView<ViewModel: GameModelProtocol>: View {
...
+                                .font(.cardSmall)
+                                .matchedGeometryEffect(id: $0.id, in: answerCardsNamespace)
                         }
                     }
-                    .padding(.trailing, 20.0)
+                    .frame(width: deckWidth)
+                    .clipShape(RoundedRectangle(cornerRadius: 8))
+                    .shadow(radius: 8)
+                    .offset(x: viewWidth - deckWidth - 20,
+                            y: geometry.size.height - 150)
```
Comment (artem-y-eleks) [inline iosApp/iosApp/Screens/Cards/CardsView.swift:88]: Would it work for small screens too?  
Like, for example, iPhone SE?

**Comment by taraspasichnyk** [comment]
Screenshot 2023-03-13 at 09 45 39
Please, update according to the design.

**Comment by RomaJZ** [comment]
> Screenshot 2023-03-13 at 09 45 39
> 
> Please, update according to the design.

It's updated (didn't update the video though)

**Comment by artem-y-eleks** [comment]
Looks good, thanks for the updates!  
I gave it a spin and, unfortunatelly, it only works once. After round 1 is completed and before round 2, the animation isn't triggered anymore. And I think we can be actually ok with that, but it block the game flow:

**Review by artem-y-eleks — APPROVED** [review]

**Pull request merged**